src/js/player.js

```diff
@@ -2525,9 +2525,9 @@ class Player extends Component {
    *
    * @param {Object} options    Options for remote text track
```

Comment:
this should be `{TextTrackList[]} theArgs track to add as remote text tracks`, also theArgs is a weird name, shouldn't it be tracks?